src/Shared/Network/Helpers/CommanderHelper.cs

```diff
@@ -32,14 +32,23 @@ public static void AddCommander(this Packet packet, ICommander commander)
 			// TODO: Assign new IDs to characters such that the social ID does not conflict.
 			packet.PutLong(commander.Id + 1);   // SocialInfoId
 
-			packet.PutInt(commander.Hp);
-			packet.PutInt(commander.MaxHp);
-			packet.PutShort(commander.Sp);
-			packet.PutShort(commander.MaxSp);
-			packet.PutInt(commander.Stamina);
-			packet.PutInt(commander.MaxStamina);
-			packet.PutShort(0); // Shield
-			packet.PutShort(0); // MaxShield
```

Comment:
I'm not sure where "MaxShield" comes from, but no such thing exists on this structure.

Reply:
We probably assumed it was another "current/max" pair, based on the others.